lib/compat/wordpress-6.4/html-api/class-wp-html-processor.php

```diff
@@ -254,7 +254,7 @@ public static function create_fragment( $html, $context = '<body>', $encoding =
 			$p->state->context_node   = array( 'BODY', array() );
 			$p->state->insertion_mode = WP_HTML_Processor_State::INSERTION_MODE_IN_BODY;
 
-			// @TODO: Create "fake" bookmarks for non-existent but implied nodes.
+			// @todo Create "fake" bookmarks for non-existent but implied nodes.
```

Comment:
Please note that these files in the HTML API are maintained in Core and backported into the Gutenberg project. Any changes will need to happen first in Core to avoid discrepancies between the codebases.

The particular changes are applied already there, I believe, and as part of the 6.4 compat layer, these files _should not_ be changed since they correspond to historic files in the 6.4 branch of WordPress.

If this is causing problems with the linting software we might consider instead finding better ways to exclude the `html-api` directory from flagging warnings.

Reply:
Thanks for the review @dmsnell. I will remove my changes for 6.4 compat layer. As well for 6.5 compat, I will keep the changes or also remove them.

Reply:
thanks @huzaifaalmesbah 

> As well for 6.5 compat, I will keep the changes or also remove them.

please note, however, that my comment only applies to the code in the `html-api` directory. since this PR has no changes in the 6.5 compat layer for the HTML API then there's nothing to update there.

Reply:
Thanks for your clarification @dmsnell 
as your changes request I remove my changes for `html-api`